Dockerfile

```diff
@@ -1,21 +1,20 @@
 FROM python:3.9-slim-buster
 
-COPY . /usr/src/tfworker
 WORKDIR /usr/src/tfworker
 
 # apt is run with 2>/dev/null to squelch apt CLI warnings and make builds look cleaner, remove to debug
 RUN apt update 2>/dev/null && \
     apt install -y --no-install-recommends \
-        wget \ 
+        wget \
         unzip && 2>/dev/null \
     wget --quiet --output-document terraform.zip https://releases.hashicorp.com/terraform/0.12.29/terraform_0.12.29_linux_amd64.zip && \
     unzip terraform.zip && \
-    rm terraform.zip && \ 
-    mv terraform /usr/local/bin && \ 
+    rm terraform.zip && \
+    mv terraform /usr/local/bin && \
     chmod 755 /usr/local/bin/terraform && \
-    python ./setup.py install && \ 
+    pip install terraform-worker && \
```

Comment:
Is this going to pull from pypi instead of the local worker?

Reply:
Yeah it didn't occur to me until talking with @ephur that the goal for the dockerfile was to run your local changes in the docker container. Updated the PR to instead build from local.